msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md

```diff
@@ -8,151 +8,25 @@ localization_priority: Normal
 keywords: teams apps meetings user participant role api  
 ---
 
-# Apps in Teams meetings
+# Apps for Teams meetings
 
-Meetings enable collaboration, partnership, informed communication, and shared feedback in an inclusive and active forum. The meeting app can deliver a user experience for each stage of the meeting lifecycle including pre-meeting, in-meeting and post-meeting app experience, depending on the attendee's status.
+Meetings enable collaboration, partnership, informed communication, and share feedback in an inclusive and active forum. The meeting app can deliver a user experience for each stage of the meeting lifecycle including pre-meeting, in-meeting and post-meeting app experience, depending on the attendee's status.
 
 Teams end-users can access apps during meetings using the tab gallery, for example:
 
 * Pre-stage a Kanban board
 * Launch an in-meeting actionable dialog
 * Create a post-meeting poll
 
-Teams’ meeting app extensibility is based on the following concepts:
+> [!VIDEO https://www.youtube-nocookie.com/embed/nKAy5rNDus4]
 
-✔ Meeting lifecycle has stages such as before, during, and after meeting time frame.  
-✔ Participant roles in a meeting such as meeting organizer, presenter, or attendee.  
-✔ User types in a meeting such as in-tenant, guest, federated, or anonymous Teams user.
+This article provides an overview of meeting app extensibility, API references, enable and configure apps for meetings, and Together Mode in Teams.
 
-This article covers information about the meeting lifecycle and how to integrate tabs, bots, and messaging extensions in your meeting. It also enables you to identify participant roles and also use different user types to perform tasks.
+You can enhance your meeting experience by using the meeting extensibility feature. This feature enables you to integrate your apps within meetings. It also includes the various stages of a meeting lifecycle where you can integrate tabs, bots, and messaging extensions. With meeting extensibility, you can identify different participant roles and user types.
```

Comment:
Reword "With meeting extensibility, you can identify different participant roles and user types."

With meetings extensibility APIs, you can identify different participant roles, user types, get meeting events, generate in-meeting dialogs, and more, which transform the meeting experience.